docs/UsersGuide/source/InputOutputFiles.rst

```diff
@@ -19,45 +20,30 @@ The external model files needed for initializing the runs can be obtained in a n
 
 Pre-processing (UFS_UTILS)
 --------------------------
-When a user runs the SRW Application as described in the quickstart guide
-:numref:`Section %s <Quickstart>`, input data for the pre-processing utilities is linked
-from a location on disk to your experiment directory by the workflow generation step. The
-pre-processing utilities use many different datasets to create grids, and to generate model
-input datasets from the external model files.  A detailed description of the input files
-for the pre-processing utilities can be found `here 
-<https://noaa-emcufs-utils.readthedocs.io/en/ufs-v2.0.0/>`_.
+When a user runs the SRW Application as described in the Quick Start Guide :numref:`Chapter %s <QuickstartC>`, :numref:`Step %s Generate the Forecast Experiment <GenerateWorkflowC>` links the input data for the pre-processing utilities from a location on disk to the experiment directory. The pre-processing utilities use many different datasets to create grids and to generate model input datasets from the external model files. A detailed description of the input files for the pre-processing utilities can be found `here <https://noaa-emcufs-utils.readthedocs.io/en/ufs-v2.0.0/>`__.
 
 UFS Weather Model
 -----------------
-The input files for the weather model include both static (fixed) files and grid and date
-specific files (terrain, initial conditions, boundary conditions, etc). The static fix files
-must be staged by the user unless you are running on a pre-configured platform, in which case
-you can link to the existing copy on that machine. See :numref:`Section %s <StaticFixFiles>`
-for more information. The static, grid, and date specific files are linked in the experiment
-directory by the workflow scripts. An extensive description of the input files for the weather
-model can be found in the `UFS Weather Model User's Guide <https://ufs-weather-model.readthedocs.io/en/ufs-v2.0.0/>`_.
-The namelists and configuration files for the SRW Application are created from templates by the
-workflow, as described in :numref:`Section %s <WorkflowTemplates>`.
+The input files for the weather model include both static (fixed) files and grid- and date-specific files (terrain, initial conditions, boundary conditions, etc). The static fix files
+must be staged by the user unless you are running on a Level 1/pre-configured platform, in which case you can link to the existing copy of the data on that machine. See :numref:`Section %s <StaticFixFiles>` for more information. The static, grid, and date-specific files are linked in the experiment directory by the workflow scripts. An extensive description of the input files for the weather model can be found in the `UFS Weather Model User's Guide <https://ufs-weather-model.readthedocs.io/en/ufs-v2.0.0/>`__. The namelists and configuration files for the SRW Application are created from templates by the workflow, as described in :numref:`Section %s <WorkflowTemplates>`.
```

Comment:
Should I just have it point to "latest" instead of v2.0.0? Same for the other UG's?

Reply:
I think ultimately these will need to be version numbers, not latest, for a tagged release. But latest can be used in develop and it can be updated in the release branch when that is created.